pytorch_lightning/trainer/training_loop.py

```diff
@@ -493,21 +479,20 @@ def run_training_epoch(self):
             if batch_output.signal == -1:
                 break
 
+            # -----------------------------------------
+            # SAVE METRICS TO LOGGERS AND PROGRESS_BAR
+            # -----------------------------------------
+            self.trainer.logger_connector.update_train_step_metrics(batch_output)
```

Comment:
@tchaton why is this moved to above `on_train_batch_end`?

`on_trrain_batch_end` can also log

Reply:
My mistake. You are entirely right !